katdal/chunkstore_rados.py

```diff
@@ -86,8 +89,13 @@ def from_config(cls, config, pool, keyring=None, timeout=5.):
         try:
             if isinstance(config, dict):
                 cluster = rados.Rados(conf=config)
-            else:
+            elif os.path.isfile(config):
                 cluster = rados.Rados(conffile=config)
```

Comment:
This is a little nasty. If some typo or screwup leads to the file not existing, then instead of a helpful exception that tells you the file wasn't found, you'll presumably get some sort of syntax error as it tries to parse the filename as a config file. I would prefer one of the following:
- add a separate parameter to take raw config text, and allow exactly one of them to be specified
- make writing a temporary file the caller's problem.

Reply:
Since RADOS is on its way out again I vote for the latter...

Reply:
Done.